### PR TITLE
Add disruption test for sequence no and primary terms

### PR DESCRIPTION
**Pull request by andreidan**
## Checklist

 - [x] User relevant changes are recorded in ``CHANGES.txt``
 - [x] Touched code is covered by tests
 - [x] Documentation has been updated if necessary
 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed
 - [x] This does not contain breaking changes, or if it does:
    - It is released within a major release
    - It is recorded in ``CHANGES.txt``
    - It was marked as deprecated in an earlier release if possible
    - You've thought about the consequences and other components are adapted
      (E.g. AdminUI)

### CODE REVIEW

**Comment by andreidan** [comment]
@seut this fixes the flakiness that was caused by a possible delayed shard allocation taking 1 minute (default) and we were waiting for a maximum of 30seconds in assertBusy. This changes the delayed time to 5 seconds (I pondered setting it to 0 but thought that having a delay might be more realistic, however I'm also ok to change it to 0)

**Comment by andreidan** [comment]
@seut fixed the flakiness further 🤞🏻

File: sql/src/test/java/io/crate/integrationtests/SQLTransportIntegrationTest.java [hunk 1]
```diff
@@ -512,6 +512,14 @@ public SQLResponse execute(String stmt, Object[] args, Session session) {
         return response;
     }
 
+    public SQLResponse execute(String stmt, @Nullable String schema, String node, Object... args) {
+        SQLOperations sqlOperations = internalCluster().getInstance(SQLOperations.class, node);
+        Session session = sqlOperations.createSession(schema, User.CRATE_USER);
```
[the remark this answers is not in the document]
Comment (andreidan) [reply sql/src/test/java/io/crate/integrationtests/SQLTransportIntegrationTest.java]: currently no. do you think we should drop the param?

[the remark this answers is not in the document]
Comment (seut) [reply sql/src/test/java/io/crate/integrationtests/SQLTransportIntegrationTest.java]: yep I'd drop it then

**Comment by andreidan** [comment]
retest this please

**Comment by andreidan** [comment]
retest this please

**Pull request merged**